verification/envs/js_node/Dockerenv

```diff
@@ -0,0 +1,5 @@
+RUN apt-get update
+RUN apt-get install -y curl
+RUN curl -sL https://deb.nodesource.com/setup | bash -
+RUN apt-get install -y nodejs
+RUN cd /opt/mission/envs/js_node/; npm install; mv /opt/mission/envs/js_node/node_modules /lib/
```

Comment:
`mv /opt/mission/envs/js_node/node_modules /lib/`
for what this line?
we can install all npm packets as global:
`npm install -g` – some like that